composer.json

```diff
@@ -29,7 +29,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Needs `composer.lock` update too - possibly good idea to drop PHP 7 to simplify the matrix

Reply:
Done